test/uk/gov/hmrc/helptosave/repo/UserCapStoreSpec.scala

```diff
@@ -50,23 +38,19 @@ class UserCapStoreSpec extends TestSupport with MongoTestSupport[UserCap, MongoU
     "getting the user-cap" should {
 
       "return the existing record successfully" in {
-
-        mockGet()(toFuture(Some(record)))
-        Await.result(mongoStore.get(), 5.seconds) shouldBe Some(record)
+        withMongo { reactiveMongoComponent ⇒
+          val store = newUserCapMongoStore(reactiveMongoComponent)
+          Await.result(store.doUpdate(record), 5.seconds)
```

Comment:
you are not asserting the return value here ?

Reply:
I've added an assert on the return value